eox_nelp/signals/tests/test_tasks.py

```diff
@@ -76,7 +85,48 @@ def test_dispatch_futurex_progress(self, post_futurex_progress_mock, generate_pr
             course_id=course_id,
             user_has_passing_grade=True,
         )
-        post_futurex_progress_mock.assert_called_with(data)
+        post_futurex_progress_mock.assert_called_with(progress_enrollment_data)
+
+    @patch("eox_nelp.signals.tasks._generate_progress_enrollment_data")
+    @patch("eox_nelp.signals.tasks._post_futurex_progress")
+    def test_not_call_dispatch_logic_setting_not_configured(
+        self, post_futurex_progress_mock, generate_progress_enrollment_data_mock
+    ):
+        """Test when course_grade_changed_progress_publisher is not called
+        due no setting configured.
+
+        Expected behavior:
+            - dispatch_futurex_progress is not called due settings.
```

Comment:
you are calling the method ...

Reply:
[5b2d1cf](https://github.com/eduNEXT/eox-nelp/pull/41/commits/5b2d1cf6d3b48ace51ae229da337b6bd82aa659b)